Cargo.toml

```diff
@@ -1,13 +1,12 @@
 [package] 
 name = "unicorn-rs"
-description = "Monster is a symbolic execution engine for 64-bit RISC-U code"
-authors = ["Alexander Lackner <monster@a.lfweb.org>", "Alexander Linz <linzalexander@gmail.com>", "Christian Mösl <christian.moesl@live.at>", "Fabian Nedoluha <monster@onders.org>", "Michael Starzinger <michael.starzinger@antforge.org>"]
-documentation = "https://docs.rs/monster"
+description = "Unicorn is a compiler and runtime system for quantum computing"
+authors = ["Michael Starzinger <michael.starzinger@antforge.org>", "Stefanie Muroya"]
 repository = "https://github.com/cksystemsgroup/unicorn"
 homepage = "https://cksystemsgroup.github.io/unicorn"
 license = "MIT"
 categories = ["command-line-utilities", "development-tools::testing", "science", "simulation"]
-keywords = ["monster", "riscv", "riscu", "symbolic-execution", "smt"]
+keywords = ["quantum annealing", "quantum computing", "bounded model checking", "symbolic-execution", "smt"]
```

Comment:
I think we should avoid spaces in the keywords, I would just us hyphens to separate the words.

See here: https://doc.rust-lang.org/cargo/reference/manifest.html#the-keywords-field

Reply:
ok! Ill change it!